build.gradle.kts

```diff
@@ -72,15 +72,24 @@ val configValues = mapOf(
 val buildDirectory: Directory = layout.buildDirectory.get()
 
 tasks.register("buildLocationsInsidePrisonApiModel", GenerateTask::class) {
-  generatorName.set("kotlin-spring")
+  generatorName.set("kotlin")
```

Comment:
Changed as kotlin-spring was generating OffsetDateTime fields when what we actually want is LocalDateTime fields.